src/scss/components/_list.scss

```diff
@@ -100,4 +109,99 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      margin-left: 0;
+
+      + & {
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .list__item {
+        width: 100%;
+      }
+    }
+  }
+
+  &--with-icons {
+    display: flex;
+    flex-grow: 1;
+
+    .list__item {
+      border-radius: 50%;
+      height: 48px;
+      margin-left: 12px;
+      text-indent: -9999px;
+      width: 48px;
+
+      a {
+        padding: 0;
+        width: 100%;
+      }
+
+      img {
+        display: block;
+        max-width: 100%;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+
+  &--with-row-icons {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+
+    .list {
+      &__item {
+        border-radius: 4px;
+        flex-grow: 1;
+
+        &::before {
+          margin-right: 0;
+        }
+
+        a {
+          display: block;
+        }
+
+        @include media(normal) {
+          width: 100%;
+        }
+
+        &:last-child {
+          margin-left: 5px;
+
+          @include media(normal) {
+            margin-left: 0;
+            margin-top: get-space(element, small);
+            max-width: initial;
```

Comment:
Can we remove this `max-width: initial`? ;)

Reply:
Removed 👍